app/models/events/new_discussion.rb

```diff
@@ -1,7 +1,8 @@
 class Events::NewDiscussion < Event
   def self.publish!(discussion)
     event = create!(kind: 'new_discussion',
-                    eventable: discussion)
+                    eventable: discussion,
+                    sequence_id: 0)
```

Comment:
My concern is that the event sequence for a discussion only makes sense for those records with a discussion_id. So if you need to create a sequence_id here.. then also add discussion_id.